Fixtures/ProducerManagerStub.php

```diff
@@ -0,0 +1,54 @@
+<?php
+declare(strict_types = 1);
+
+namespace M6Web\Bundle\KafkaBundle\Fixtures;
+
+use M6Web\Bundle\KafkaBundle\AbstractManager;
+use M6Web\Bundle\KafkaBundle\Producer\ProducerManager;
+
+/**
+ * Class ProducerManagerStub
+ * @package M6Web\Bundle\KafkaBundle\Fixtures
+ *
+ * A stub class to imitate ProducerManager class
+ */
+class ProducerManagerStub extends ProducerManager
+{
+    /**
+     * @param \RdKafka $entity
+     * @return AbstractManager
+     */
+    public function setEntity(\RdKafka $entity): AbstractManager
+    {
+        return $this;
+    }
+
+    /**
+     * @param string $brokers
+     * @return AbstractManager
+     */
+    public function addBrokers(string $brokers): AbstractManager
+    {
+        return $this;
+    }
+
+    /**
+     * @param int $logLevel
+     * @return AbstractManager
+     */
+    public function setLogLevel(int $logLevel): AbstractManager
+    {
+        return $this;
+    }
+
+    /**
+     * @param string             $name
+     * @param \RdKafka\TopicConf $rdKafkaTopicConf
+     * @param array|null         $confToSet
+     * @return $this
```

Comment:
Je suis assez d'accord sur le fait qu'un code explicite doit pouvoir se lire sans commentaire. Personnellement, j'ai l'impression de faillir à ma mission quand j'insère un commentaire. 
Mais c'est par là que plusieurs apprécient d'appréhender un code et comme la qualité est l'affaire d'une équipe ...
So as you (all) want